tests/system/providers/amazon/aws/example_batch.py

```diff
@@ -94,6 +97,14 @@ def create_job_queue(job_compute_environment_name, job_queue_name):
     )
 
 
+@task(trigger_rule=TriggerRule.ONE_FAILED)
+def get_describe_job(job_id):
```

Comment:
```suggestion
def describe_job(job_id):
```
Will need to also update the call to this method below of course